420.js

```diff
@@ -0,0 +1 @@
+console.log("O")
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.dev.codacy.org/gh/troubleshoot-codacy/test-js/pullRequest?prid=35331)
```suggestion
console.log("O");
```